financials/reports/funding_transparency_2020Q4.md

```diff
@@ -0,0 +1,67 @@
+# 2020Q4 Funding Transparency Report
+
+Last updated: Mar 14 2021
+
+## Introduction
+This document provides a high level overview of Grin's financials for Q4 of 2020 (Oct 01 - Dec 31). The purpose is to keep the community up to date on funding activity in transparent and digestible way. The data is derived from what is reported in the financial logs under [/grin-pm/financials](https://github.com/mimblewimble/grin-pm/tree/master/financials).
+
+### Notes
+
+1. Data was up to date at the time of writing, but it's still possible that there have been changes and updates in the logs since.
+2. USD conversion rates are derived from the closing price of the asset on the day in question according to Coingecko.com unless otherwise stated. 
+
+## Status before the period
+
+### Disposable funds
+
+At the end of Sep 30 2020, Grin held disposable funds to the equivalent of $1,127,130.25 in the following assets:
+
+Currency | Amount | USD x-rate Sep 30 2020 | USD Equivalent | Wallet address(es)
+|---|---:|---:|---:|---|
+GRIN | 3770.71555489 | $0.306694 | $1,156.46 | https://donations.grin-tech.org
+BTC | 104.514006 | $10,784.49 | $1,127,375.92 | 39hgKkhfPFpwKRQ6yARGwU2uuVay69NKu4<br />bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx
+| | | | $1,127,130.25 |
+
+### Additional details
+* See the [2020Q3 report](funding_transparency_2020Q3.md).
+
+## Activity during the period
+
+### Received
+
+During Oct 01 - Dec 30 2020, Grin received the following assets: 
+
+Currency | Amount | Wallet address(es)
+|---|---:|---|
+GRIN | 0 | https://council-donations.yeastplume.org
+BTC | 0 | bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx <br/> 39hgKkhfPFpwKRQ6yARGwU2uuVay69NKu4
+
+### Spent
+
+During Oct 01 - Dec 30 2020, Grin spent the following assets:
+
+Currency | Amount | Tx fees | Total amount |
+|---|---:|---:|---:|
+BTC|7.638498|0.002864|7.641362
+
+### Comments
+* Tx outputs were split into smaller 10BTC chunks in this period. Aside from the transaction fees involved in doing so, all spending in the period was related to funding requests, specifically:
```

Comment:
Where is Antioch's Q4 2020 payment? I don't see it here or on the Q3 report: https://github.com/mimblewimble/grin-pm/blob/5d9578efe2d889d84293f4c1f99ce7ce6f926544/financials/reports/funding_transparency_2020Q3.md#activity-during-the-period

Reply:
It looks like your Q4 payment is not listed either.

Reply:
Thanks for raising, was included in the numbers, but was missed in the call outs. Fixed.